src/utilities/gql.ts

```diff
@@ -496,7 +496,7 @@ const gql = {
 
   DELETE_PLAN_SNAPSHOT: `#graphql
     mutation DeletePlanSnapshot($snapshot_id: Int!) {
-      deletePlanSnapshot: delete_plan_snapshot(snapshot_id: $snapshot_id) {
+      deletePlanSnapshot: delete_plan_snapshot_by_pk(snapshot_id: $snapshot_id) {
```

Comment:
do you mind updating the permissions util file to check this specific query as well? (sorry for the repeat comment, not sure if it showed up if I commented directly on the commit)